packages/itinerary-body/src/styled.tsx

```diff
@@ -571,15 +558,22 @@ export const StepDescriptionContainer = styled.div`
 `;
 
 export const StepsHeaderAndMapLink = styled.span`
-  display: inline-block;
+  display: inline-flex;
   align-self: center;
   margin-top: 10px;
+
+  a {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
 `;
 
 export const StepsHeader = styled(TransparentButton)`
   color: #676767;
   font-size: 13px;
   font-style: normal;
+  width: 100%;
```

Comment:
Revert this change, it makes for an odd layout (see screenshot)

<img width="266" alt="image" src="https://user-images.githubusercontent.com/56846598/228886479-09f7588f-c945-40cd-ade3-50824e157564.png">

Reply:
I'm not quite what this transparent button is or why we added it, but it's causing a lot of problems

Reply:
Clicking the button expands or collapses the turn-by-turn directions.